README.md

```diff
@@ -10,15 +10,58 @@ None
 
 | variable | description | default |
 |----------|-------------|---------|
-
+| `jenkins_slave_user` | Jenkins user | `{{ __jenkins_slave_user }}` |
+| `jenkins_slave_group` | Jenkins group | `{{ __jenkins_slave_group }}` |
+| `jenkins_slave_home` | Jenkins home | `{{ __jenkins_slave_home }}` |
+| `jenkins_slave_package` | package name | `{{ __jenkins_slave_package }}` |
```

Comment:
also, in `README`, please replace a placeholder with something meaningful. a simple description is enough like "Configure jenkins slave". That description should be placed in the repository description (see the top page of the repository and you will find "No description, website, or topics provided.").

```
A brief description of the role goes here
```